GVFS/GVFS.Installer.Mac/scripts/Distribution.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <pkg-ref id="com.git.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+            GIT_CHOICE_OUTLINE_PLACEHOLDER
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="VFSFORGIT_VERSION_PLACHOLDER" onConclusion="none">VFSFORGIT_PKG_NAME_PLACEHOLDER</pkg-ref>
+    GIT_CHOICE_ID_PLACEHOLDER
+    GIT_PKG_REF_PLACEHOLDER
+    <installation-check script="InstallationCheck()"/>
+    <script>
+        function InstallationCheck(prefix) 
+        {
+            var blockingProcesses = GetBlockingProcesses();
+            if (blockingProcesses.length > 0)
+            {
+                my.result.message = "Quit these processes and try again - " + blockingProcesses.join(", ") + ".";
+                my.result.type = 'Fatal';
+                return false;
+            }
+            
+            return true;
+        }
+        
+        function GetBlockingProcesses()
+        {
+            var watchList = [ "gvfs", "GVFS.Mount", "git", "gitk", "Wish" ];
+            var blockingProcesses = new Array();
+            for (var process of watchList)
+            {
+                var isRunningCmd = "[ `ps -ef | grep -w '" + process + "' | grep -v grep | wc -l` -eq 0 ]";
```

Comment:
Would the output of `ps -Ac -o command` not be easier/less error-prone to parse than `ps -ef`? Also, `grep`'s exit code is 0 if one or more lines were matched, and 1 if not, so you shouldn't need to use `wc`.

Reply:
`ps -Ac -o command` is definitely better. Keeping `wc` - without it, looks like the following `-eq 0` check uses the output of `grep -v grep` which will be always `0` and breaks our logic.

Reply:
I don’t think you need the `grep -v grep` step anymore as the command arguments aren’t part of the output when using `-c`.

Reply:
That works, Thank you!